routes/wishlists.router.js

```diff
@@ -0,0 +1,77 @@
+const express = require("express");
+const router = express.Router();
+const { Wishlist } = require("../models/wishlist.model");
+const { User } = require("../models/user.model");
+
+
+router.param("userid", async(req, res, next, id)=>{
```

Comment:
I have seen this code getting repoeated, could we create a middleware to check this?
Just a thought

Reply:
Yes, I will refactor it while doing JWT